### ISSUE TITLE
Add discussion time start and duration server message

### ISSUE THREAD

**Issue by GoogleFrog** [ZeroK-RTS/Zero-K-Infrastructure#2488]
I want to disable the start button during the discussion time. Ideally I would also disable the button during map voting.

**Comment by DeinFreund** [comment]
How about a message to indicate when you can't start polls? That'd be true for both discussion time and when there are other polls. Would you like to know specifically what reason is blocking polls?

**Comment by Licho1** [comment]
I have a feeling this all becomes to over engineered and complicated. Let people choose what they want to do where they want to play, no need to social engineer everything

**Comment by DeinFreund** [comment]
I suppose you're talking about the streamlined hosts in general because this specific change would only improve the UI by showing the user when he can and can't start polls.

**Comment by Licho1** [comment]
Yes I dont like them much .. its a full circle.. we abandoned those things in favor of player controlled places. And now we are moving back to more authoritarian regime

**Comment by DeinFreund** [comment]
@Licho1 this is the alternative to forcibly limiting big rooms to smaller player counts. I'd much rather give people a poll to go in a smaller room than alienate the big teams players. From a recent discussion in #zkadmin: 

> Let's try limiting rooms to about 22-26 people if DeinFreund's invitations don't work

I'd also believe that even though there is an added time for the small teams vote, having the map vote with multiple options is a lot faster than the old map vote spam. With a few tweaks, like letting users nominate maps or using the map ratings, it might even be better at finding the maps people want to play on (votespam didn't do that, it was just about who called !map faster). 

If people don't like the outcome of the system, they can still manually call all commands as they are used to. 

@GoogleFrog please tell me exactly how you want the message to look. Should it tell you whether to enable the start button or should it tell you about the discussion time? Whether you can or can't use the start button depends on factors outside of discussion time and polls, such as being the room owner.

**Comment by GoogleFrog** [comment]
That is a very complicated system. I am not even sure what I want to display anymore.

**Comment by DeinFreund** [comment]
If you want the server can simply keep the lobby updated if `!start` is going to do something or just display an error. Then you know when to display the button.
